### DISCUSSION TITLE
Add WGDashboard to the WireGuard container

### DISCUSSION THREAD

**Discussion by Def3nder** [tteck/Proxmox#81] [Ideas]
Hi,

what about adding WGDashboard to the WireGuard container ?

This way there would be a UI to configure the tunnels.

I would love to get this integrated in your wonderfull script 😄 

Thx a lot,
Ralf

**Comment by tteck** [comment]
Very good idea!! DONE!

**Reply by Def3nder** [reply]
super-cool ! Thx a lot !

**Comment by titof2375** [comment]
we can not put us user through the web interface?
